src/frameworks/ez.md

```diff
@@ -1,21 +1,146 @@
+To do so, uncomment the `redissession` entry in the `.platform/services.yaml` file and the corresponding relationship
+in the `.platform.app.yaml` file.  The bridge code that is provided with eZ Platform 1.13 and later will automatically
+detect the additional Redis service and use it for session storage.
 
 ## Modifying an existing eZ Platform project
 
-If you have an existing eZ Platform project that was upgraded from a previous version, or want to resynchronize with the latest recommended configuration, please see the [eZ Platform official repository](https://github.com/ezsystems/ezplatform).
+If you have an existing eZ Platform project that was upgraded from a previous version, or want to resynchronize with
+the latest recommended configuration, please see the 
+[eZ Platform official repository](https://github.com/ezsystems/ezplatform).
 
 In particular, see:
  
- * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
+ * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which
+ automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
  * The [.platform](https://github.com/ezsystems/ezplatform/tree/master/.platform) directory
- * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically enable Redis-based cache and session support if detected.
+ * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration
+  file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically 
+  enable Redis-based cache and session support if detected.
+
+
+## Local Development with eZ Platform 2.x and later
+
+eZ systems provide a tool called [eZ Launchpad](https://ezsystems.github.io/launchpad/) for local development that
+improves Developer eXperience and reduces the container management. eZ Launchpad is ready to work with Platform.sh.
+
+### eZ Launchpad installation
+
+eZ Launchpad approach is to stay as decoupled as possible from your development machine and your remote hosting whether
+you are Linux or Mac OSX. Installation is really simple:
+
+```bash
+curl -LSs https://ezsystems.github.io/launchpad/install_curl.bash | bash
+```
+ 
+Then you can just start to use it to initialize your eZ Platform project on top docker.
+
+```bash
+~/ez init
+```
+
+or just create the docker stack based on an existing project
+
+```bash
+git clone yourproject.git application; cd application; ~/ez create
```

Comment:
Break into multiple lines, please, for clarity.

Reply:
done